gobblin-kubernetes/gobblin-service/dev-cluster/gaas-application.conf

```diff
@@ -0,0 +1,73 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Sample configuration properties for the Gobblin Service
+
+# Topology Catalog and Store
+gobblin.service.work.dir=/tmp/gobblin-as-service
+
+# TopologySpec Factory
+topologySpec.store.dir=${gobblin.service.work.dir}/topologySpecStore
+topologySpecFactory.topologyNames=localGobblinCluster
+topologySpecFactory.localGobblinCluster.description="StandaloneClusterTopology"
+topologySpecFactory.localGobblinCluster.version="1"
+topologySpecFactory.localGobblinCluster.uri="gobblinCluster"
+topologySpecFactory.localGobblinCluster.specExecutorInstance.class="org.apache.gobblin.runtime.spec_executorInstance.LocalFsSpecExecutor"
+topologySpecFactory.localGobblinCluster.specExecInstance.capabilities="source:dest"
+topologySpecFactory.localGobblinCluster.gobblin.cluster.localSpecProducer.dir=${gobblin.service.work.dir}/jobs
+
+# Flow Catalog and Store
+flowSpec.store.dir=${gobblin.service.work.dir}/flowSpecStore
+
+# Template Catalog
+gobblin.service.templateCatalogs.fullyQualifiedPath="file://"
+
+# JobStatusMonitor
+gobblin.service.jobStatusMonitor.enabled=false
+
+# FsJobStatusRetriever
+fsJobStatusRetriever.state.store.dir=${gobblin.service.work.dir}/state-store
+
+# DagManager
+gobblin.service.dagManager.enabled=true
+gobblin.service.dagManager.jobStatusRetriever.class="org.apache.gobblin.service.monitoring.FsJobStatusRetriever"
+gobblin.service.dagManager.dagStateStoreClass="org.apache.gobblin.service.modules.orchestration.FSDagStateStore"
+gobblin.service.dagManager.dagStateStoreDir=${gobblin.service.work.dir}/dagStateStoreDir
+
+# RestLI
+gobblin.service.port=6956
+
+# MySQL State Store
```

Comment:
if `dev` is built around file systems, do we need `mysql` environment here? A user can pick either `dev` suite or `mysql` suite.

Reply:
Ah I should rename it to be better then, `dev` was originally intended for `mysql` plus whatever job communication mechanism we are relying on, which is `fs`. The only difference between `dev` and `base` would be the mysql store, so I guess it would be better to rename it to mysql-cluster